src/components/ColorInput.jsx

```diff
@@ -1,9 +1,11 @@
 import React from "react"
-
-const ColorInput = ({ inputId, bgColor, onChange }) => {
+import {AiOutlineCloseCircle} from 'react-icons/ai'
+const ColorInput = ({ inputId, bgColor, onChange,handleDelete }) => {
+  console.log(typeof inputId)
   return (
     <div className="colorInputContainer">
       <div className="colorSwatch" style={{ backgroundColor: bgColor }}></div>
+      {(inputId > 2)?<AiOutlineCloseCircle color="red" className="deleteButton" onClick={()=>handleDelete(inputId-1)}/>:null}
```

Comment:
The icon itself only renders as an SVG and to be accessible to keyboard users it needs to wrapped in a button element. The icon itself would preferably be grey instead of red, so it doesn't stand out too much and with a white background so the input box border isn't visible through it. 

I also wonder about the logic. To me it would make more sense if when there are more than 2 inputs then all of them have a delete button so I can delete whichever color I wish, however when it gets down to only 2 inputs left then the buttons disappear. So instead of checking for inputId > 2 we check that the array length is > 2 instead.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix